authn/service.go

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package authn
+
+import (
+	"context"
+	"errors"
```

Comment:
You can use `"github.com/mainflux/mainflux/errors"` here

Reply:
@blokovi Yes, that's the plan. There is a separate issue for that: #970, and I'll resolve it ASAP.